build/BuildSetupTemplate.js

```diff
@@ -20,7 +20,7 @@ const packageJsonContents = fs.readFileSync(path.join(__dirname, "..", "package.
 const packageMeta = JSON.parse(packageJsonContents)
 const { version, name } = packageMeta
 const prodName = packageMeta.build.productName
-const pathVariable = "{app}\\resources\\app\\cli\\windows\\"
+const pathVariable = "{app}\\resources\\app\\cli\\win\\"
```

Comment:
I swapped this to take advantage of the `${os}` variable in `electron_builder`, such that we only include the CLI script for the needed platform.